<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>396</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>659</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cmboxAvtemCompr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Human readable = 23:233[R251 G144 B012]s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Human readable smart = 32:122[253,144,13]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Human readable hex = 20:7A[FD,90,D]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BinarySimple RGBA = •NULL#s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BinaryLoop = •NULL#s•NULL# 128 3byteInt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BinaryLoopGen2 = •NULL#0x31•F#0xFF0x30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AVIM2 = ONE MORE COLOR MAP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AVIM3  = colMaps + repeatCount</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="vlabel_13">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>431</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>740</width>
      <height>358</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="1">
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>colorMaps took:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="labelColorMapsPerc">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>--%</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>17</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btnOpenTheir">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Open image</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>1 byte indexes took:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label1ByteIndexesPerc">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>-%</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="labelRepeatCountsPers">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>--%</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>repeatCounts took:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="btnOpenAimg">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Open AvtemImage</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="btnSaveToFile">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Save as AvtemImage</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="1">
               <widget class="QLabel" name="labelPixelCount">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>---</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="vlabel_18">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Pixel count:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="labelNonComprSize">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>130</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>---</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="vlabel_19">
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>CF: x--</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="labelWidthHeight">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>--x--</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="labelSizeOnDiskTheir">
                  <property name="text">
                   <string>-- bytes</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelCompMultiplierTheir">
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>CF: x--</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="labelColRR">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="vlabel1">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Color count:</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="vlabel_17">
                <property name="maximumSize">
                 <size>
                  <width>175</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Size on disk (Avtem compression):</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="vlabel0">
                <property name="text">
                 <string>colRR</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="labelSizeOnDiskAvtem">
                  <property name="maximumSize">
                   <size>
                    <width>130</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-- bytes</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelCompMultiplierAvtem">
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>CF =</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="vlabel_16">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Width x Height:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="labelColorCount">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>--</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLabel" name="labelBPColorAvtem">
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="vlabel3">
                <property name="text">
                 <string>Size on disk (Their compression):</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="vlabel_15">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>File size (non-compressed)</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="vlabel4">
                <property name="text">
                 <string>bytes per color:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelimg">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>PIcture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
